examples/creatives/banner_nonlinear.js

```diff
@@ -0,0 +1,51 @@
+class BannerNonLinear extends BaseSimidCreative{
```

Comment:
I think its fine for this PR, but on a future PR you should add some comments to this class to explain what it does for developers.

Reply:
Got it, will handle in future pr